tests/continental_extension_mixed_compositions_particles.prm

```diff
@@ -0,0 +1,345 @@
+# This is a simplified version of the continental extension cookbook to test
+# the combination of different compositional field discretizations and
+# particles
+
+####  Global parameters
+set Dimension                              = 2
+set Start time                             = 0
+set End time                               = 30000
+set Use years in output instead of seconds = true
+set Nonlinear solver scheme                = iterated Advection and Stokes
+set Nonlinear solver tolerance             = 1e-5
+set Max nonlinear iterations               = 5
```

Comment:
```suggestion
set Max nonlinear iterations               = 2
```

This change could speed up the test a bit without dropping any feature coverage.